content/blog/2016/2016-05-19-update-plugin-for-pipeline-tutorial.md

```diff
@@ -0,0 +1,189 @@
+---
+:layout: post
+:title: Refactoring a Jenkins plugin for compatibility with Pipeline jobs
+:tags:
+- core
```

Comment:
Makes sense to add "pipeline" and "plugin development" tags. BTW. I do not know a right tag for "plugin development". @rtyler WDYT?